notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<blockquote>Americas:&nbsp;<code>us-central1</code></blockquote><blockquote>Europe:&nbsp;<code>europe-west4</code></blockquote><blockquote>Asia Pacific:&nbsp;<code>asia-east1</code></blockquote><p> </p><p>Is that up to date? Suggest linking to the locations page instead of listing the locations here (as this list might get stale)</p><p> </p><p>Suggest: "You can change the <code>REGION</code> variable, which specifies where all operations and resources are ran and created. We recommend that you choose the region closest to you. For more information about which features are available in each region, see <a href="https://cloud.google.com/vertex-ai/docs/general/locations" target="_blank">Vertex AI regions</a>.</p><p> </p><p>You cannot use a multi-regional bucket for training with Vertex AI." (Is this sentence necessary here? Seems out of place. Can it be moved to where it's needed?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>